poutyne/framework/callbacks/lr_scheduler.py

```diff
@@ -12,28 +12,22 @@
 from .callbacks import Callback
 
 
-class _PyTorchLRSchedulerWrapper(Callback):
-    def __init__(self, torch_lr_scheduler, *args, **kwargs):
+class LRScheduler(Callback):
```

Comment:
I don't really see the value of having two classes here (LRScheduler and _PyTorchLRSchedulerWrapper). Both refer to a scheduler attribute that works like a PyTorch's LR scheduler. I would merge these two classes.

Reply:
Yes I think it came as a good idea in the first place, but since all the optimizers and schedulers we can use with Poutyne will have to be instance of torch.optim and torch.optim.lr_scheduler, we can safely assume the strong Pytorch dependance and remove the useless parent class.

Reply:
Fixed by 11ac150dfce64b15b7f8aa26677ba2ba42581fcd